package.json

```diff
@@ -8,7 +8,7 @@
     "url": "git+https://github.com/apollostack/subscriptions-transport-ws.git"
   },
   "dependencies": {
-    "@types/node": "^6.0.38",
```

Comment:
It seemed strange that when pulling the project downstream that it was installing this as a dependency. I noticed and moved since I was adding the backo2 module. I can revert that change if desired.